glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -261,7 +262,7 @@ open class GleanInternalAPI internal constructor () {
             val originalEnabled = getUploadEnabled()
 
             @Suppress("EXPERIMENTAL_API_USAGE")
-            Dispatchers.API.launch {
+            Dispatchers.API.executeTask {
```

Comment:
This needs to happen before any of the preinit tasks run.  `executeTask` merely adds to the "real" queue rather than the preinit queue.  After preinit time, it's basically identical to `launch`.